docs/tutorials/multimodality.md

```diff
@@ -29,17 +29,17 @@ For audio/video-to-text tasks, AI DIAL has adapters for Gemini 1.5 Pro and Flash
 
 You can use [DIAL SDK](https://github.com/epam/ai-dial-sdk) to create custom applications, which are basically any custom logic with a conversation interface packaged as a ready-to-use solution. Refer to [Tutorials](../tutorials/quick-start-with-application) to learn how to create a simple application or watch a [demo video](../video%20demos/demos-for-developers/dial-develop-and-deploy).
 
-Such application can be designed and configured to use multi-modal LLMs to perform specific tasks or even form an ecosystem of applications that can interact with each other. 
+Such application can be designed and configured to use multimodal LLMs to perform specific tasks or even form an ecosystem of applications that can interact with each other. 
 
-In the Cookbook section, you can view several examples:
+In the Cookbook section, you can find several examples:
 
 * [Text-to-text](../Cookbook/dial-cookbook/examples/how_to_call_text_to_text_applications)
 * [Text-to-image](../Cookbook/dial-cookbook/examples/how_to_call_text_to_image_applications)
 * [Image-to-text](../Cookbook/dial-cookbook/examples/how_to_call_image_to_text_applications)
 
 ### Orchestrator
 
-Besides applications, you can create a comprehensive solution that can combine several applications and models to perform more complex and interconnected tasks. Such solutions can be called orchestrators. 
+Besides creating an application which solves a specific multimodal task, you could create a generic application which is aware of multimodal DIAL models and is able to use them as tools to solve a given task. We call such generic applications orchestrators. 
 
 [DIAL ChatHub](../video%20demos/demos/dial-chathub) is an example of a flow orchestrator that combines several applications and models into one unified access point. ChatHub can automatically route prompts to one of several agents (text-to-text applications, text-to-image applications, vision-to-text applications) depending on the task that needs to be performed. For example, if a user asks about weather, the Web RAG agent is engaged, if a user wants to output an image based on the text input - a specific application handles this task that is connected with a corresponding model. All this is done while interacting with one ChatHub solution.
```

Comment:
What does "flow" mean in "flow orchestrator"?
It looks like a word salad if no clear definition is provided.